<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PakageShowForm</class>
 <widget class="QDialog" name="PakageShowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_id"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>快递单号</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>订单号</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>411</width>
     <height>183</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>寄件地址</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>收件地址</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="textEdit_sendaddr"/>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="textEdit_recieveaddr"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_order">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>default</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>快递公司</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>寄件时间</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_sendtime"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>收件人电话</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_recievephone"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_company">
      <item>
       <property name="text">
        <string>中通</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>圆通</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>申通</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>韵达</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>中国邮政</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>京东</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>顺丰</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>极兔</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>快递单</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>寄件人</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_sendname"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>寄件人电话</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_sendphone"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>收件人</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_recievename"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_submit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>创建快递单</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
